.github/workflows/ci.yaml

```diff
@@ -28,7 +28,7 @@ jobs:
           go-version: '1.21'
 
       - name: Test
-        run: go test -v ./...
+        run: go test ./...
```

Comment:
I don't think that you have to dig through lots of output should be that big of a deal. Granted Go makes it hard to see when test output begins, but we put in work to separate the pieces. Just because the action you're using as an impetus for this PR didn't require verbose output to see the problem doesn't mean other people's test failures don't. I don't think the logs are that egregious.

Reply:
Update: To clarify, since verbose is automatically applied on the test that fails, I can be ok with this change. I would like to hear others' opinion here real quick before approving though.

Reply:
I'm only proposing this for that exact reason: failing tests will always be dumped verbosely :)